asmcomp/amd64/proc.ml

```diff
@@ -309,7 +334,10 @@ let max_register_pressure = function
         if fp then [| 3; 0 |] else  [| 4; 0 |]
   | Iintop(Idiv | Imod) | Iintop_imm((Idiv | Imod), _) ->
     if fp then [| 10; 16 |] else [| 11; 16 |]
```

Comment:
This is completely orthogonal to the present pull request, but just for the record: these `if fp` are rather ugly and should eventually be handled along the lines of `num_destroyed_by_plt_stub`, i.e. with a `num_allocatable_int_regs` constant that is 13 or 12 depending on `fp`.

Reply:
Also, I noticed that some of these are inaccurate as regards Spacetime, which sometimes occupies another hard register.  I can try to fix these all at once.

Reply:
In the end these "max pressure" numbers are heuristics for preventive spilling, so no wrong code will be generated if the numbers are off.  But, yes, there is room for improvement here, both in precision of the numbers and in elegance of the code!